docs/ReleaseHistory.md

```diff
@@ -11,11 +11,12 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
-# 1.16.0 - 03/03/2025
+# 1.16.0 - 03/04/2025
 - BRK: Eliminate `SEC000/101.Unclassified32CharacterString` as noisy and not useful.
 - BRK: Rename `SEC101/102.AdoPat` friendly name to `AdoLegacyPat`.
 - BUG: Correct `SEC000/002.Unclassified16ByteHexadecimalString` id and rule name  on calling `GetMatchIdAndName` (where `SEC000/001.Unclassified64ByteBase64String` was returned incorrectly before).
 - BUG: Resolve `System.FormatException: The input is not a valid Base-46 string` errors calling `SEC101/102.AdoPat.GetMatchIdAndName` by swallowing correct exception kind `ArgumentException` in `IsChecksumValid` helper.
+- BUG: `?P<name>` is now used throughout for named captures as this is required currently for RE2 compatibility. This reverts a clean-up in this area (that broke RE2) in 1.15.0.
```

Comment:
RE2 was broken before this cleanup since most rules did not have the ?P before I touched anything.

Reply:
This is more than a revert, in other words.

Reply:
Hiding my earlier messages on this and starting over since I was being personally defensive while being less than clear about my actual concern with the release note.

I believe this is true:

* 1.14.0 and earlier: All but a handful of patterns had RE2-*in*compatible syntax already. 
* 1.15.0: All patterns have RE2-*in*compatible syntax
* 1.16.0 (proposed): All patterns have RE2 compatible syntax.

If so, I don't think this release note quite captures the change.